### ISSUE TITLE
Cannot get globalTags

### ISSUE THREAD

**Issue by erkyrath** [y-lohse/inkjs#38]
In Lectrote, I am trying to get the globalTags to extract the game title, but I get an exception:

> Uncaught TypeError: Cannot read property 'every' of undefined
>    at i.value (/Users/zarf/src/lectrote/inkjs/ink.min.js:3:7808)
>    at i.get (/Users/zarf/src/lectrote/inkjs/ink.min.js:3:12111)
>    at <anonymous>:1:6

The code is just

    story = new Story(str);
    console.log(story.globalTags);

**Comment by erkyrath** [comment]
The file in question (compiled  with inky 0.6.1):

[test.ink.txt](https://github.com/y-lohse/inkjs/files/530690/test.ink.txt)
[test.json.txt](https://github.com/y-lohse/inkjs/files/530691/test.json.txt)

**Comment by y-lohse** [comment]
Hm, there's no glaring msitake, so it must be a logic problem somewhere. 

Pinging @joethefish for some insight: In [TagsAtStartOfFlowContainerWithPathString](https://github.com/y-lohse/inkjs/blob/master/engine/Story.js#L1162), `flowContainer` is the root container, and then `innerWeaveContainer` becomes the first object in that container. In this case it's a Tag... which is not a container and hence has no `content` property. `innerWeaveContainer` should obviously point to a container, but which..?

**Comment by joethephish** [comment]
Argh, apologies, yeah, this is my bad - think it's a bug related to me inlining containers in the runtime.

In my defence, I just discovered that Xamarin hasn't been running all my unit tests due to silently hitting an internal error and not reporting it. God knows what else has broken without me noticing...

**Comment by joethephish** [comment]
Referencing https://github.com/inkle/ink/issues/210, which needs to be done before any fixes can be done, sigh.

**Comment by joethephish** [comment]
C# fix implemented [here](https://github.com/inkle/ink/issues/209), and will be releasing a bumped ink version very soon.

**Comment by y-lohse** [comment]
I've updated inkjs with the changes as well. Glad to see nothing else broke due to the new structure ^^
